functions/lib/emails/email_reminder.py

```
@@ -12,127 +12,198 @@
-
-  return https_fn.Response(f"Sent all reminders for upcoming events.")
\ No newline at end of file
+    uid = str(uuid.uuid4())
+    logger = Logger(f"email_reminder_logger_{uid}")
+    logger.add_tag("uuid", uid)
+
+    tomorrow = (datetime.now(SYDNEY_TIMEZONE) + timedelta(days=1)).date()
+
+    logger.info(
+        "Sending reminder emails for events for date "
+        + tomorrow.strftime("%d/%m/%Y, %H:%M:%S")
```

Ruff (0.14.0)

Logging statement uses `+`

(G003)